setup.py

```diff
@@ -133,7 +133,7 @@ def extra(self):
         'typing-extensions>=4.2.0'
     ],
     extras_require={
-        'email': ['email-validator>=1.0.3'],
+        'email': ['email-validator>=2.0.0.post2'],
```

Comment:
is this right? I thought we are supporting older `email-validator`?

Reply:
I guess it's just `extra`, so we could do this, but I think it's more confusing that leaving it more open.

Reply:
```suggestion
        'email': ['email-validator>=1.0.3'],
```